mods/e2140/content/ed/infantry/a02/rules.yaml

```
@@ -0,0 +1,41 @@
+ed_infantry_a02:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A02 Android
+	Valued:
+		Cost: 75
+	Buildable:
+		IconPalette:
+		Queue: Infantry.ED
+	Health:
+		HP: 28
+	RevealsShroud:
+		Range: 4c896
+	Armament@PRIMARY:
+		Name: primary
+		Weapon: ed_infantry_a02
+		LocalOffset: 100,0,150
+		MuzzlePalette:
+		PauseOnCondition: !ammo
+	Armament@SECONDARY:
+		Name: secondary
+		Weapon: ed_infantry_a02_air
+		LocalOffset: 100,0,150
+		MuzzlePalette:
+		PauseOnCondition: !ammo
+	AmmoPool:
+		Ammo: 2
+		AmmoCondition: ammo
+	ReloadAmmoPool:
+		Delay: 62
+		ResetOnFire: True
+		Count: 1
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 2
+		Animation: DATABASE/A02.FLC
+		Title: A02 Android
```

Use original name *Android A02*:

```suggestion
		Title: Android A02
```